packages/react-router-dom/docs/api/Prompt.md

```diff
@@ -1,3 +1,3 @@
 # &lt;Prompt>
 
-TODO
+Re-exported from core [`Prompt`](../../../react-router/docs/api/Prompt.md)
```

Comment:
Yes, this is correct. `react-router-native` re-exports everything from core

https://github.com/ReactTraining/react-router/blob/a757d7f2c56bfbc54833a82620a67f8ddf83e7e7/packages/react-router-native/main.js#L1